modules/test/playwright/tests/dynamic-data-mapping-form-web/formEntries.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {formsPagesTest} from '../../fixtures/formsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {deleteItems} from './utils/deleteItems';
+
+export const test = mergeTests(loginTest(), formsPagesTest);
+
+test.afterEach(async ({formsPage, page}) => {
+	await formsPage.goTo();
+
+	await deleteItems(formsPage, page);
+});
+
+test.beforeEach(({page}) => {
+	page.setViewportSize({height: 1080, width: 1920});
+});
+
+test('can interact with a large list of fields on the form entries page', async ({
+	context,
+	formBuilderPage,
+	formBuilderSidePanelPage,
+	page,
+}) => {
+	await formBuilderPage.goToNew();
+
+	for (const index of Array.from(Array(30).keys())) {
+		await formBuilderSidePanelPage.addFieldByDoubleClick('Text');
+
+		await formBuilderSidePanelPage.label.fill(`Text ${index}`);
+
+		await formBuilderSidePanelPage.clickBackButton();
+	}
+
+	await formBuilderPage.clickSaveButton();
+
+	await formBuilderPage.publishButton.click();
```

Comment:
hey @igor-franca, testing this locally, I think we don't need those two steps, considering that publishing will automatically save it, and that inside openFormSubmissionMethod() method , we are already publishing the forms as well. So this alone should cover the steps We need to achieve the form submission page, what do you think? You can test it as well and let me know!